packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -403,17 +403,22 @@ export default class PciStoragesContainersService {
       });
   }
 
-  deleteHighPerfObject(projectId, containerId, objectKey) {
+  deleteHighPerfObject(projectId, containerId, region, objectKey) {
```

Comment:
i think it's better to implement it at the end of function parameters, to avoid breaking change.
If this function is used in others place it break.

```suggestion
  deleteHighPerfObject(projectId, containerId, objectKey, region) {
```